ja-odur/puzzled
# #167453425 create sudoku grid

**What does this PR do?**
* create sudoku grid

**Description of Task to be completed?**
* create sudoku grid

**How should this be manually tested?**
* Pull and checkout to this branch locally by running `git fetch origin ft-create-sudoku-grid-167453425 && ft-create-sudoku-grid-167453425`
* Start  the virtual environment `pipenv shell`
* install python requirements `pipenv install`
* install node requirements `npm install`
* migrate the database `python manage.py migrate`
* run builds `npm run build`
* start the django server `python manage.py runserver`
* Hit the endpoint `http://127.0.0.1:8000/` and click on the first sudoku card, you see a grid.

**Any background context you want to provide?**
* N/A

**What are the relevant pivotal tracker stories?**
* [167453425 ](https://www.pivotaltracker.com/story/show/167453425)
